Package.swift

```diff
@@ -30,7 +30,7 @@ import PackageDescription
 #elseif os(Linux)
 	
 	let CryptoLibUrl = "https://github.com/IBM-Swift/OpenSSL.git"
-    let CryptoLibVersion: Version = "1.0.0"
+    let CryptoLibVersion: Package.Dependency.Requirement = .upToNextMajor(from: "1.0.1")
```

Comment:
There's nothing wrong with this, though it's an unnecessary change (compiler already infers this type information), but the syntax is now inconsistent with the macOS equivalent above.

You could change it to just `Version = "1.0.1"`, or change both to have the more explicit type information.  The explicit type information is helpful when wanting to test locally with a branch (which was your case while developing this fix), so perhaps that's a good reason to apply it to both (ie. change the macOS line 28 to `Package.Dependency.Requirement = .upToNextMajor(from: "1.0.0")`